extensions/src/test/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessorTest.java

```diff
@@ -176,7 +177,7 @@ public void get003WriteFiletest() throws Exception {
       //When an unknown host URL is used with an import, the private method writeFile() is called.
       CommandLineProcessor.main( unknownHostUrl );
       try {
-        assertEquals( exception, FileUtils.readFileToString( file ) );
+        assertEquals( exception, FileUtils.readFileToString( file , StandardCharsets.UTF_8) );
```

Comment:
remove the space after "file" argument and leave space between UTF_8 and close paranthesis